.sync/rust_config/rustfmt.toml

```diff
@@ -5,7 +5,7 @@
 
 # Keep these options sorted in ascending order to ease lookup with rustfmt documentation.
 
-edition = "2021"                 # This would normally be picked up from Cargo.toml if not specified here
+edition = "2024"                 # This would normally be picked up from Cargo.toml if not specified here

```

Comment:
Let's just delete it to keep things simple.

```suggestion
```

Reply:
It will fall back to Cargo.toml. Which I think is fine for Mu repos. We don't have so much Mu Rust code that tracking fmt edition updates separately seems necessary.